src/components/KInput/KInput.vue

```diff
@@ -225,34 +246,38 @@ export default defineComponent({
 
     watch(value, (newVal, oldVal) => {
       if (newVal !== oldVal) {
-        handleInput({ target: { value: newVal } })
+        updateInputValue(String(newVal))
```

Comment:
@sumimakito I'm wondering if this is where the KInput behavior inadvertently changed?

Reply:
@adamdehaven Yes. This should be the line that caused this change.